app/platform/platform.c

```diff
@@ -879,7 +879,7 @@ uint32_t platform_s_flash_write( const void *from, uint32_t toaddr, uint32_t siz
   if(SPI_FLASH_RESULT_OK == r)
     return size;
   else{
-    NODE_ERR( "ERROR in flash_write: r=%d at %08X\n", ( int )r, ( unsigned )toaddr);
+    NODE_ERR( "ERROR in flash_write: r=%d at %p\n", ( int )r, ( unsigned )toaddr);
```

Comment:
We actually have %p support in the printf? I thought we didn't (but I could be getting myself confused with one of the other platforms I work on).

Reply:
It's fairly recent, but still already there:  #2062